docs/website/docs/plus/features/projects.md

```diff
@@ -241,7 +324,30 @@ pipelines:
     destination: filesystem
     dataset_name: bronze
```

Comment:
I actually think you can get rid of this yaml example, it doesn't make sense to be here.